crates/workspace/src/pane.rs

```diff
@@ -2190,7 +2195,9 @@ impl Pane {
                             .read(cx)
                             .item_for_entry(entry, cx)
                             .and_then(|item| item.project_path(cx))
-                            .map(|project_path| project_path.path);
+                            .map(|project_path| project_path.path)
+                            .map_or(None, |path| if path.exists() { Some(path) } else { None });
```

Comment:
I followed it down to this https://github.com/zed-industries/zed/blob/74223c1b009662840de04fde6bfcc9ba4780fddf/crates/project/src/worktree_store.rs#L148-L159

I am not familiar how `Worktree`s work yet, but it seems that even for a single out-of-project file `Worktree` is created with the same `Path`. Meaning this `Path::strip_prefix()` will nuke whole path and we'll be left with empty string. 

I considered checking paths here for equality and returning `None` if they're equal. This code is pretty low level and I don't know if this would be correct thing to do.